waltid-services/waltid-e2e-tests/src/test/kotlin/ExchangeExternalSignaturesApi.kt

```diff
@@ -1,26 +1,37 @@
+import COSE.AlgorithmID
 import E2ETestWebService.loadResource
+import cbor.Cbor
+import com.nimbusds.jose.jwk.ECKey
+import id.walt.crypto.keys.KeyType
 import id.walt.crypto.keys.jwk.JWKKey
 import id.walt.did.utils.randomUUID
+import id.walt.issuer.issuance.IssuanceExamples
 import id.walt.issuer.issuance.IssuanceRequest
+import id.walt.issuer.issuance.IssuanceType
+import id.walt.mdoc.COSECryptoProviderKeyInfo
+import id.walt.mdoc.SimpleCOSECryptoProvider
+import id.walt.mdoc.dataelement.*
+import id.walt.oid4vc.data.AuthenticationMethod
+import id.walt.oid4vc.data.ProofType
+import id.walt.sdjwt.utils.Base64Utils.encodeToBase64Url
 import id.walt.webwallet.db.models.WalletCredential
 import id.walt.webwallet.db.models.WalletDid
+import id.walt.webwallet.service.exchange.IssuanceService
 import id.walt.webwallet.service.keys.SingleKeyResponse
-import id.walt.webwallet.web.controllers.exchange.PrepareOID4VPRequest
-import id.walt.webwallet.web.controllers.exchange.PrepareOID4VPResponse
-import id.walt.webwallet.web.controllers.exchange.SubmitOID4VPRequest
+import id.walt.webwallet.web.controllers.exchange.*
 import id.walt.webwallet.web.model.EmailAccountRequest
 import io.ktor.client.*
 import io.ktor.client.call.*
 import io.ktor.client.request.*
-import io.ktor.client.statement.*
 import io.ktor.http.*
 import io.ktor.server.util.*
 import kotlinx.coroutines.runBlocking
-import kotlinx.serialization.json.Json
-import kotlinx.serialization.json.contentOrNull
-import kotlinx.serialization.json.jsonPrimitive
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.decodeFromByteArray
+import kotlinx.serialization.json.*
 import kotlinx.uuid.UUID
-import kotlin.test.assertFalse
+import kotlin.test.assertNotEquals
+import kotlin.test.assertNotNull
 
 class ExchangeExternalSignatures {
```

Comment:
not part of this pr, but just noticed
since this test suite wipes the wallet data before running the tests,
maybe it's worth mentioning that in e2e test, at this location:
```kotlin
// test External Signature API Endpoints
ExchangeExternalSignatures().executeTestCases()
```